### DISCUSSION TITLE
Banking down from #0 does not work

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#67] [Editor App Feature Requests] [answered]
Using down arrow from Bank 0 does not take me to bank 99. Bank # is just stuck at 0.

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by konnsim** [comment, marked as answer]
Fixed in latest version.
